python/lsst/daf/butler/datastores/inMemoryDatastore.py

```diff
@@ -91,6 +87,15 @@ class InMemoryDatastore(Datastore):
     """A new datastore is created every time and datasets disappear when
     the process shuts down."""
 
+    storageClassFactory: StorageClassFactory
+    """Factory for creating storage class instances from name."""
+
+    datasets: Dict[int, object]
```

Comment:
You've used `Any` instead of `object` elsewhere (as have I), though I think they are indeed equivalent here.